apps/biobin/resources/usr/bin/biobin-summary.py-20170726-r4221

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+import sys
+import argparse
+import glob
+
+if __name__ == "__main__":
+
+    parser=argparse.ArgumentParser()
+
+    parser.add_argument("-p", "--prefix", help="Prefix of BioBin files (do not include the '-')", required=True)
+    parser.add_argument("-c", "--control-value", help="Phenotype control value", default="0")
+    parser.add_argument("-a", "--all-control", help="Use all non-missing subjects as controls", action="store_true", default=False)
+    parser.add_argument("-s", "--sep", help="Separator character in bins file(s)", default=",")
+    parser.add_argument("-o", "--output-sep", help="Separator character for output", default="\t")
+
+    args=parser.parse_args()
```

Comment:
recommend using a code formatter here (e.g. black https://github.com/ambv/black) to make everything match the style guide (e.g. should be spaces around '=')